cmd/kueue/main.go

```diff
@@ -153,6 +156,26 @@ func main() {
 		SecureServing:  true,
 		FilterProvider: filters.WithAuthenticationAndAuthorization,
 	}
+
+	if cfg.InternalCertManagement == nil || !*cfg.InternalCertManagement.Enable {
+		metricsCertPath := "/etc/kueue/metrics-certs"
+		setupLog.Info("Initializing metrics certificate watcher using provided certificates",
+			"metrics-cert-path", metricsCertPath)
+
+		var err error
+		metricsCertWatcher, err := certwatcher.New(
+			filepath.Join(metricsCertPath, "tls.crt"),
+			filepath.Join(metricsCertPath, "tls.key"),
```

Comment:
It would be better to add this fixed named certs name in the installation guide.

Reply:
Yea, let me file an issue for documentation on this. @mimowo and I are thinking we would do a follow up PR with documentation showing an example of how to enable this.

My first draft of this I included a values file to enable this but that is probably not the way we want to go.

Reply:
Sounds good. I will file a future issue for this as I think the documentation of this is important.

Addressed the enable api field in the latest commit.

Reply:
SGTM